imagescan/subscriber.go

```diff
@@ -270,3 +285,28 @@ func (s *Subscriber) getScanMode(img *image) string {
 	}
 	return mode
 }
+
+func (s *Subscriber) sendImageOwnerChanges(images []*image) {
+	for _, img := range images {
+		if img.ownerChanges.empty() {
+			continue
+		}
+
+		func(img *image) {
```

Comment:
if we are not handling error anyway and seems there's no shared data, why not run as goroutine?

Reply:
Updated